src/graphx/graphx.asm

```diff
@@ -4156,29 +4156,18 @@ gfx_RotateSpriteHalf:
 ;  arg1 : Pointer to sprite struct output
 	ld	iy,0
 	add	iy,sp
-	push	ix
-	ld	ix,(iy+3)
-	ld	c,(ix+0)		; c = width
-	ld	b,(ix+1)		; b = height
-	lea	de,ix+2
-	ld	ix,(iy+6)
-	ld	(ix+0),c
-	ld	(ix+1),b
+	ld	hl,(iy+3)
+	ld	bc,(hl)		; c = width ; b = height
+	inc	hl
+	ld	iy,(iy+6)
+	ld	(iy+0),bc
 	mlt	bc
-	lea	hl,ix+1
 	add	hl,bc
-	push	ix
-.loop:
-	ld	a,(de)
-	ld	(hl),a
+	lea	de,iy
+ploop:	inc	de
```

Comment:
Why was this label changed? Local labels are preferred inside of routines like this, and it's also preferred for labels to exist on their own line.

Reply:
ah, I found the "." weird (actually, I realised later fasmg requires it ).
Please revert to ".loop" ;)

Reply:
@mateoconlechuga 
Thanks to confirm my doubt ;)
then, I suggest to insert between lines 4166-4167 a "push de" and between lines 4170-4171 a "pop hl" . I think it will be okay :)